studio/local_db_provider.py

```diff
@@ -0,0 +1,51 @@
+import os
+import json
```

Comment:
Database low-level logic implemented simply as directory with file names being the keys
and json values serialized into files themselves.